src/custom-elements/src/utils/Drag.ts

```diff
@@ -0,0 +1,87 @@
+let self: Drag;
+
+export class Drag {
+  private trigger: HTMLElement;
+  private elementToMove: HTMLElement;
+  private initialTransformStyling: string;
+  private pos1: number;
+  private pos2: number;
+  private pos3: number;
+  private pos4: number;
+  private component: any;
+
+  constructor(trigger: HTMLElement, elementToMove: HTMLElement, component: any) {
+    self = this;
+    this.trigger = trigger;
+    this.elementToMove = elementToMove;
+    this.component = component;
+    this.trigger.addEventListener('mousedown', (e) => this.dragMouseDown(e));
+    this.trigger.addEventListener('touchstart', (e) => this.dragTouch(e));
```

Comment:
You are adding anonymous functions to the listeners `mousedown` and `touchstart`. You will not be able to remove them later.